src/Core/Types/ControlType.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Maybe remove this file from the PR since the only remaining change is the byte order marker?